netfile_client/NetFileClient.py

```diff
@@ -195,7 +195,7 @@ def get_auth(self, env_file):
     def fetch(self, endpoint, **kwargs):
         """ Fetch all of a particular record type """
         url = self._base_url + getattr(Routes, endpoint)
-        params = self._params
+        params = { **self._params }
```

Comment:
I've never seen ** used inside braces.  Did you test the behavior here?  If it's what you want, then I guess it's ok, but a comment about what's happening may help.

Reply:
It's a pretty standard usage. It gets [first mention in the docs](https://docs.python.org/3/reference/expressions.html#dictionary-displays), before unpacking into function kwargs, if that's the usage you've seen before.

> A double asterisk ** denotes dictionary unpacking. Its operand must be a [mapping](https://docs.python.org/3/glossary.html#term-mapping). Each mapping item is added to the new dictionary. Later values replace values already set by earlier dict items and earlier dictionary unpackings.

It allows a more concise syntax to do the same as `copy()` and `update()`
```python
item = {
  'name': 'apple',
  'price': .50,
  'department': 'produce',
  'category': 'fruit'
}
item2 = {
  **item,
  'name': 'banana',
  'price': 1.0
}
```